lib/apiservers/engine/backends/archive_test.go

```diff
@@ -15,12 +15,15 @@
 package backends
 
 import (
+	"context"
 	"testing"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 
 	"github.com/docker/docker/api/types"
```

Comment:
goimports/gofmt: there should be three import groups for packages: standard, third-party, and own.

There are four here.